src/Components/bookHub/BookHub.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
+import Card from './BookCard';
+import SearchBar from './SearchBar';
+
+// airtable configuration
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+function CardsDisplay() {
+  const [cards, setCards] = useState([]);
```

Comment:
Can you rename this state to something more connotative? Like 'books' and 'setBooks' the naming clearly conveys what's being stored

```suggestion
  const [books, setBooks] = useState([]);
```